btr-api/tests/postman/btr-api.postman_collection.json

```diff
@@ -213,23 +213,23 @@
 								},
 								{
 									"key": "Date",
-									"value": "Wed, 13 Dec 2023 21:40:11 GMT"
+									"value": "Fri, 15 Dec 2023 18:21:32 GMT"
```

Comment:
Shouldnt we b e using dates in ISO format ?

Reply:
yep you're right -- think I just need to run the example again

Reply:
oh wait nvm this is just in the header value (automatic from postman)